notebooks/SpecvizExample.ipynb

```diff
@@ -75,17 +76,9 @@
    "source": [
     "## Load a file to Specviz\n",
     "\n",
-    "Here we download a JWST spectrum from [MAST](https://masttest.stsci.edu/) using `astroquery`. \n",
-    "By default the downloaded file goes to your temp directory, and thus may eventually be \n",
-    "garbage collected (deleted) by your system. If you would like to have the file permanently, \n",
-    "simply uncomment the second line below and provide the directory path on your system where \n",
-    "you would like the file stored.\n",
+    "We will download an example JWST spectrum using `astropy`. The downloaded file can be found in the '/Users/username/.astropy/cache/download/' folder.\n",
     "\n",
-    "One other thing to note about retrieving MAST data through astroquery is that it caches\n",
-    "the data by default. It is possible for files to be updated in MAST with more recent calibrations\n",
-    "but remain the same size, in which case your local cached file would not be replaced by the new\n",
-    "version. You can turn off caching by setting `cache=False` in the `download_file` call to\n",
-    "force it to re-download the file if desired."
+    "Data can also be downloaded from [MAST](https://masttest.stsci.edu/) using `astroquery`. Please see other notebooks for examples."
```

Comment:
This is the "TEST VERSION" of MAST – don't we want https://mast.stsci.edu/ instead?